django/core/mail/message.py

```diff
@@ -169,7 +171,10 @@ def __init__(self, _text, _subtype='plain', _charset=None):
             # We do it manually and trigger re-encoding of the payload.
             MIMEText.__init__(self, _text, _subtype, None)
             del self['Content-Transfer-Encoding']
-            self.set_payload(_text, utf8_charset)
+            has_long_lines = any(len(l) > 998 for l in _text.splitlines())
```

Comment:
maybe a module constant could make this slightly more readable? e.g. `RFC5322_EMAIL_LINE_LENGTH_LIMIT = 998`